src/pages/index.md

```diff
@@ -7,30 +7,44 @@ layout: ~/layouts/Main.astro
 
 ## What is Astro?
 
-**Astro** is a _fresh but familiar_ approach to building websites. Astro combines decades of proven performance best practices with the DX improvements of the component-oriented era.
+**Astro** is a _fresh but familiar_ approach to building websites. Astro combines proven performance and best practices along with modern DX improvements from the new component-oriented ESM era.
 
-With Astro, you can use your favorite JavaScript framework and automatically ship the bare-minimum amount of JavaScript—by default, it's none at all!
+With Astro, you can use your favourite JavaScript framework and automatically ship the bare-minimum amount of JavaScript — by default, which is preferably none at all!
 
 ## Project Status
 
-⚠️ **Astro is still an early beta, missing features and bugs are to be expected!** If you can stomach it, then Astro-built sites are production ready and several production websites built with Astro already exist in the wild. We will update this note once we get closer to a stable, v1.0 release.
+⚠️ **Astro is still an early beta,**
```

Comment:
This section was intentionally left as a single paragraph so that it didn't take up too much of the page and distract from the overall job of documenting snowpack. I'm not opposed to smaller changes here, but lets move this section back into a single paragraph.

Reply:
Will do =) 
Means we get rid of colin the caterpillar from the page...if you are bored google colin the caterpillar,